Ticket - 51948 Advantzware TEST

sewa67 opened the pull request:
Please review this change and merge into development branch 
Files - est/copypack.p
dEstPacking.w
est/CopyEnhancPack.i New File

sewa67 committed:
Ticket - 51948 Advantzware TEST
Files - est/copypack.p,dEstPacking.w,est/CopyEnhancPack.i

brad-advantzware reviewed (changes requested):
dEstPacking.w - Change the list of supported material types to a global variable (constant) with INITIAL value of "C,5,6,M,D"
Change CopyEnhancPack.i to estCopyEnhancedPack.p and pass parameters (rowids)
Move the "bf-estPacking.estPackingID = NEXT-VALUE(estPackingID_seq) into the CREATE trigger of estPacking.p (find all instances of this and remove, once the create trigger is set.)

sewa67 committed:
Ticket - 51948 Advantzware TEST
Files - create.trg/estPacking.p,est/EstPacking.w,est/copypack.p,est/dEstPacking.w,est/CopyEnhancedPack.p

brad-advantzware reviewed (changes requested):
Sewa - The comment block at the top of CopyEnhancedPack.p was copied and pasted and is inaccurate.  You should have the comment block that gets auto created from PDSOE.  For a sample of that, see any of the new Procs.p programs (CostProcs.p).  Commenting in code is an area that we need to improve on.

sewa67 committed:
Ticket - 51948 Advantzware TEST
Files - est/CopyEnhancedPack.p
